test/parallel/test-console-clear.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const stdout_write = process.stdout.write;
+
+// The sequence for moving the cursor to 0,0 and clearing screen down
+const check = '\u001b[1;1H\u001b[0J';
+
+let buf = '';
+
+// Fake TTY
+process.stdout.isTTY = true;
+
+process.stdout.write = common.mustCall((string) => buf += string, 2);
```

Comment:
Nit: it doesn't harm, but I find `common.mustCall()` redundant in this case. The assertion below is sufficient imo.